packages/yoroi-extension/app/ergo-connector/components/signin/CardanoSignTxPage.js

```diff
@@ -250,17 +284,95 @@ class SignTxPage extends Component<Props> {
     );
   }
 
+  renderAddresses(): Node {
+    const addresses = this.props.txData.outputs.map(({ address }) =>  address )
+    return (
+      <div className={styles.toAddresses}>
+        {addresses.map((address, idx) => {
```

Comment:
@AhmedIbrahim336 please note that this is buggy. If, for example, `addresses` has 3 elements, then it shows the first address and `2 more` *twice*.

Reply:
It's fixed by 1931a18d3785689fba87a462fe2e7d0a82606701.